README.md

```diff
@@ -1,7 +1,9 @@
-# curriculum-databases-projects-template
+# Vet Clinic
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+> This template is used to practice SQL commands 
+
```

Comment:
Great job @mohamed20163858 🤝

- [ ] [Optional] Kindly fix the MIT license link. At the moment it appears broken. This will make your readme file more professional.